src/main/java/com/stripe/model/Charge.java

```diff
@@ -887,6 +895,647 @@ public void setRuleObject(Rule expandableObject) {
+      @SerializedName("bank_name")
+      String bankName;
+
+      /** Two-letter ISO code representing the country the bank account is located in. */
+      @SerializedName("country")
+      String country;
+
+      /**
+       * Uniquely identifies this particular bank account. You can use this attribute to check
+       * whether two bank accounts are the same.
+       */
+      @SerializedName("fingerprint")
+      String fingerprint;
+
+      /** Last four digits of the bank account number. */
+      @SerializedName("last4")
+      String last4;
+
+      /** Routing transit number of the bank account. */
+      @SerializedName("routing_number")
+      String routingNumber;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Alipay extends StripeObject {}
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Bancontact extends StripeObject {
+      /** Bank code of bank associated with the bank account. */
+      @SerializedName("bank_code")
+      String bankCode;
+
+      /** Name of the bank associated with the bank account. */
+      @SerializedName("bank_name")
+      String bankName;
+
+      /** Bank Identifier Code of the bank associated with the bank account. */
+      @SerializedName("bic")
+      String bic;
+
+      /** Last four characters of the IBAN. */
+      @SerializedName("iban_last4")
+      String ibanLast4;
+
+      /**
+       * Preferred language of the Bancontact authorization page that the customer is redirected to.
+       * Can be one of `en`, `de`, `fr`, or `nl`.
+       */
+      @SerializedName("preferred_language")
+      String preferredLanguage;
+
+      /**
+       * Owner's verified full name. Values are verified or provided by Bancontact directly (if
+       * supported) at the time of authorization or settlement. They cannot be set or mutated.
+       */
+      @SerializedName("verified_name")
+      String verifiedName;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Bitcoin extends StripeObject {
+      @SerializedName("address")
+      String address;
+
+      @SerializedName("amount")
+      Long amount;
+
+      @SerializedName("amount_charged")
+      Long amountCharged;
+
+      @SerializedName("amount_received")
+      Long amountReceived;
+
+      @SerializedName("amount_returned")
+      Long amountReturned;
+
+      @SerializedName("refund_address")
+      String refundAddress;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Card extends StripeObject {
+      /**
+       * Card brand. Can be `amex`, `diners`, `discover`, `jcb`, `mastercard`, `unionpay`, `visa`,
+       * or `unknown`.
+       */
+      @SerializedName("brand")
+      String brand;
+
+      /** Check results by Card networks on Card address and CVC at time of payment. */
+      @SerializedName("checks")
+      Checks checks;
```

Comment:
Okay I saw the commend in my other PR too. I'll fix mine to just duplicate the class instead of trying to be smart.

Reply:
@remi-stripe, I would prefer not sharing in this example, simply because it is what the back-end actually has. So I stated before that now we prefer sharing, I'm actually wrong. I think I was saying that because in most case backend code do share resources. So my position is rather resource sharing just as what our backend and open API has.

In the world that we share nothing, that's when we can ignore the backend code and openAPI references (with exceptions of sharinging core resource)